apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/CustomIdGenerator.kt

```diff
@@ -0,0 +1,24 @@
+package com.apollographql.apollo.compiler
+
+interface CustomIdGenerator {
+  fun apply(
+      /**
+       * The minified query string source with fragments
```

Comment:
nit: maybe it's better to say `GraphQL operation document to be sent to the server`